build.gradle.kts

```diff
@@ -12,6 +12,7 @@ val springDocVersion = "1.6.9"
 dependencies {
   implementation("org.springframework.boot:spring-boot-starter-webflux")
   implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+  implementation("org.flywaydb:flyway-core")
```

Comment:
Just a comment about migrations - is there any way we can timestamp these `i.e. yyyymmddhhmmdd`, rather than prefixing with `V1` etc, I find this keeps the files order nicely, rather than `V1, V10, V11` etc

Reply:
Sure, have removed the V prefix - so can be named like 20220803__migration_title.sql now